src/requests/getImages.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if(!query) {
+    return Promise.resolve([])
+  } else {
+    return axios
+      .get(`https://images-api.nasa.gov/search?q=${query}`)
+      .then((response) => {
+        const imageResults = (response.data.collection.items)
+
+// ORIGINAL PARSE RESULTS METHOD         
```

Comment:
You dont need to include previous iterations to show you you got where you ended up, thats one of the main things version control is for. You can delete all this commented out code